_pages/plugins/adipoq.md

```diff
@@ -0,0 +1,28 @@
+---
+mediawiki: AdipoQ
```

Comment:
The `mediawiki` tag is for backlinking to the old content from the MediaWiki-based ImageJ wiki. Since this is new content, it should not be used here.